meta-arm/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -0,0 +1 @@
+require ${@bb.utils.contains('MACHINE_FEATURES', 'uefi-secureboot', 'systemd-uefi-secureboot.inc', '', d)}
```

Comment:
@mikkorapeli-linaro

> I think this should be machine feature "efi", and this should possibly go to poky
> upstream systemd recipe.
>
> Then, it's possible to switch from grub to systemd-boot with uki binaries
> but that requires some more work. There both kernel and initrd will be signed
> into an efi binary. Only problem for me there is the split between different
> layers: poky, meta-arm, meta-security and meta-secure-core. Not clear which
> bits would belong where, and which layer would contain the test build config
> and oeqa tests.

There are several changes here that are outside the scope of the [initial task](https://linaro.atlassian.net/browse/CAS-236), which does not include the use of uki binaries and systemd-boot. I am not against it, but it's something we will need to discuss further.

And yes, there are also places where these changes should be, because there is already meta-secure-core adding UEFI Secure Boot, but only for x86; ARM is not supported.